CMakeLists.txt

```diff
@@ -39,13 +112,15 @@ execute_process(
 
 # }}}
 
-link_directories(${PYOPENCL_CL_LIB_DIRS})
+link_directories(${CL_LIB_DIR})
+include_directories(${CL_INC_DIR} ${Python_NumPy_INCLUDE_DIRS})
 
 find_package(nanobind CONFIG REQUIRED)
-find_package(NumPy REQUIRED)
-find_package(OpenCL REQUIRED)
 
-include_directories(${OpenCL_INCLUDE_DIR} ${NumPy_INCLUDE_DIRS})
+set(OpenCL_ROOT ${CL_LIB_DIR})
+set(OpenCL_INCLUDE_DIR ${CL_INC_DIR})
+set(OpenCL_LIBRARY ${CL_LIBNAME})
```

Comment:
Doesn't `find_package(OpenCL)` set `OpenCL_LIBRARY` as an output?

Reply:
The effect is that it constrains FindOpenCL below to the options selected by the user.

Reply:
Ah, that's cool! I thought `find_library(OpenCL_LIBRARY ...)` would overwrite the variable, but apparently it just skips the search. Thanks!